Initial version of ebclfsa container

MofX opened the pull request:
# Changes

 - split cmake presets, to allow limiting the available configurations for ebclfsa apps (e.g. no x86_64)
 - Add layer with tools for ebclfsa development
   - musl: for high integrity application (will change in the future, just for demo purpose now)
   - lisa-elf-enabler: Required to mark high integrity applications for the linux kernel
   - cmake presets and toolchain for musl (ebclfsa-aarch54)
 
_(Provide a brief description of the changes in this pull request)_
# Dependencies:
 lisa-eld-enabler debian package has to be downloaded and provided manually -> to be changed, once the package is available
# Tests results
 N/A
# Developer Checklist:
- [ ] Test: Changes are tested
- [ ] Doc: Documentation has been updated 
- [ ] Git: Informative git commit message(s)
- [ ] Issue: If a related GitHub issue exists, linking is done

## Checklists for documentation
- [ ] Grammar: the content is grammatically correct
      (spelling, grammar, formatting, US English is used)
- [ ] Comprehensibility/Unambiguousness: the content is easy readable, clear to understand
- [ ] Correctness and consistency: the content is technically correct and consistent,
      no contradictions, no double descriptions
- [ ] Terminology: technical terms are clear and they are used correctly and documented in the glossary
- [ ] Level of detail: the content matches the detail level necessary for the reviewed object
      
# Reviewer checklist:
- [x] Review: Changes are reviewed
- [x] Review: Tested by the reviewer
      
## Checklists for documentation
- [ ] Grammar: the content is grammatically correct
      (spelling, grammar, formatting, US English is used)
- [ ] Comprehensibility/Unambiguousness: the content is easy readable, clear to understand
- [ ] Correctness and consistency: the content is technically correct and consistent,
      no contradictions, no double descriptions
- [ ] Terminology: technical terms are clear and they are used correctly and documented in the glossary
- [ ] Level of detail: the content matches the detail level necessary for the reviewed object

antonjahn commented on layers/embdgen/Dockerfile:
```diff
@@ -38,7 +38,7 @@ RUN pip install jsonpickle robotframework requests pyyaml psutil
 # Install EBcl build tools
 WORKDIR /build
-RUN git clone --branch v1.3.3 https://github.com/Elektrobit/ebcl_build_tools ebcl_build_tools
+RUN git clone --branch hypervisor_tooling https://github.com/Elektrobit/ebcl_build_tools ebcl_build_tools
```
```suggestion
RUN git clone --branch v1.3.4 https://github.com/Elektrobit/ebcl_build_tools ebcl_build_tools
```

antonjahn reviewed (commented):
looks good, but we keep this PR back until we can get lisa-elf-enabler_1.0-4.10_amd64.deb  from linux.elektrobit.com

MofX commented:
@antonjahn
- Dropped download of apt key, it is already added to the trusted keys in the base layer
- Moved repo definition to a lists-file to keep the same pattern as the base layer
- Combined the install of musl and lisa-elf-enabler, to reduce build time and improve readability

antonjahn commented on layers/ebclfsa/conf/apt/ebclfsa.list:
```diff
@@ -0,0 +1 @@
+deb https://linux.elektrobit.com/ebclfsa_2024.12/ ebclfsa dev
```
Please switch to https://linux.elektrobit.com/eb-corbos-linux/1.4 ebclfsa dev

antonjahn commented on layers/embdgen/Dockerfile:
```diff
@@ -38,7 +38,7 @@ RUN pip install jsonpickle robotframework requests pyyaml psutil
 # Install EBcl build tools
 WORKDIR /build
-RUN git clone --branch v1.3.3 https://github.com/Elektrobit/ebcl_build_tools ebcl_build_tools
+RUN git clone --branch v1.3.4 https://github.com/Elektrobit/ebcl_build_tools ebcl_build_tools
```
Please bump to 1.3.5
